_stbt/core.py

```diff
@@ -665,6 +665,27 @@ def test_motionresult_repr():
             "frame=<1280x720x3>)")
 
 
+class IsScreenBlackResult(object):
+    """The result from `is_screen_black`.
+
+    :ivar Frame frame: The video frame that was analyzed.
+
+    :ivar bool result: True if the screen was black. This is the same as
+        evaluating the ``IsScreenBlackResult`` as a bool.
+    """
+    def __init__(self, frame, result):
+        self.frame = frame
+        self.result = result
+
+    def __nonzero__(self):
+        return self.result
+
+    def __repr__(self):
+        return ("IsScreenBlackResult(frame=<%s>, result=%r)" % (
```

Comment:
I think result should be first here - it's what we usually care about.

Perhaps it could be called `is_black` or `black` to be more descriptive too?  `MotionResult` has `.motion`.  `MatchResult` has `.match`.

Reply:
Done.